va/va_trace.c

```diff
@@ -1773,10 +1773,10 @@ static void va_TraceVAIQMatrixBufferMPEG2(
     va_TraceMsg(trace_ctx,"\tload_non_intra_quantiser_matrix = %d\n", p->load_non_intra_quantiser_matrix);
     va_TraceMsg(trace_ctx,"\tload_chroma_intra_quantiser_matrix = %d\n", p->load_chroma_intra_quantiser_matrix);
     va_TraceMsg(trace_ctx,"\tload_chroma_non_intra_quantiser_matrix = %d\n", p->load_chroma_non_intra_quantiser_matrix);
-    va_TraceMsg(trace_ctx,"\tintra_quantiser_matrix = %d\n", p->intra_quantiser_matrix);
```

Comment:
suppose the code should be if(load_intra_quantiser_matrix){for(int i = 0; i < 64; i ++){if(i %8 == 0)va_TraceMsg(trace_ctx,"\tintra_quantiser_matrix = ", p->intra_quantiser_matrix);va_TraceMsg(trace_ctx,"\tintra_quantiser_matrix = %d", p->intra_quantiser_matrix[i];if(i % 8 == 7) va_TraceMsg(trace_ctx,"\n",}